app/assets/javascripts/mobile/mobile_comments.js

```diff
@@ -11,6 +11,8 @@
     initialize: function() {
       var self = this;
 
+      new Diaspora.MarkdownEditor(".comment_box");
```

Comment:
Maybe it is out of scope for this PR, but are we okay with the object design where constructor actually performs some job besides the object creation? This decision was made before this PR, and if it was fine back then, then these calls are fine either.

Perhaps it would be a little cleaner to have something like
```js
new Diaspora.MarkdownEditor().renderOn(".comment_box");
```

Perhaps it is a kind of nit picking, but I felt I have to notice that.